tests/_fixtures/chronos/job.json

```diff
@@ -0,0 +1,76 @@
+{
+  "id": "/foo",
+  "description": "Foo Description",
+  "labels": {
+    "name": "foo",
+    "project": "metronome",
+    "stage": "test"
+  },
+  "schedule": {
```

Comment:
In the latest design doc iteration this has now become an array, `schedules`. Example:

```
"schedules": [  // Optional, if ?embed=schedules is specified
    {
      "id": "everyBirthday",
      "description": "lorem ipsum",
      "cron": "0 19 6 9 *",
      "timezone": "Europe/Berlin",
      "startingDeadlineSeconds" : 60,
      "concurrencyPolicy": "allow",
      "enabled": true
    }
  ],

```